backend/iot_mobile/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.3'
```

Comment:
docker-compose.yaml is in the project root directory. Why do we need a second one in backend section?